src/main/groovy/org/checkerframework/gradle/plugin/CheckerFrameworkPlugin.groovy

```diff
@@ -318,7 +318,8 @@ final class CheckerFrameworkPlugin implements Plugin<Project> {
         }
       } catch (Exception e) {
```

Comment:
In particular, the exception thrown here in the usual case is that we try to read the version out of the Jar file that gradle has already loaded as a dependency, but that throws if there is no such dependency.

Reply:
I kept the debug logging in place because it might be useful later - the code above is a bit iffy - but because the failure case is common I hid it behind the debug flag